ios/FluentUI.Demo/FluentUI.Demo/Demos/DateTimePickerDemoController.swift

```diff
@@ -164,3 +195,33 @@ extension DateTimePickerDemoController: DateTimePickerDelegate {
         return true
     }
 }
+
+class DemoBarButtonItems: NSObject {
```

Comment:
not really sure we really need this class at all

Reply:
Removed class 'DemoBarButtonItems'

Reply:
Removed the class 'DemoBarButtonItems'